tradesman/model_creation/add_country_borders.py

```diff
@@ -1,33 +1,79 @@
+import geopandas as gpd
+import pandas as pd
+import urllib.request
+from os.path import join, isfile
+from tempfile import gettempdir
 import pycountry
 import requests
 from shapely.geometry import Polygon, MultiPolygon
 
 
-def get_borders_online(country_name: str):
-    country_code = pycountry.countries.search_fuzzy(country_name)[0].alpha_3
+def get_borders_gadm(model_place: str):
+    """
+    Import country borders from GADM.
+    Parameters:
+         *model_place*(:obj:`str`): current model place
+    """
+    country_code = pycountry.countries.search_fuzzy(model_place)[0].alpha_3
+
+    url = f"https://geodata.ucdavis.edu/gadm/gadm4.1/gpkg/gadm41_{country_code}.gpkg"
+
+    dest_path = join(gettempdir(), f"gadm_{country_code}.gpkg")
+    if not isfile(dest_path):
+        urllib.request.urlretrieve(url, dest_path)
+
+    level0 = gpd.read_file(dest_path, layer="ADM_ADM_0")
+    level0 = level0.assign(level=0, division_name="country_border")
+    level0["geom"] = gpd.GeoSeries.to_wkb(level0.geometry)
+    level0.rename(columns={"COUNTRY": "country"}, inplace=True)
+
+    return level0.geom.values[0]
+
+
+def get_borders_online(model_place: str):
+    """
+    Import country borders from GeoBoundaries.
+    Parameters:
+         *model_place*(:obj:`str`): current model place
+    """
+    country_code = pycountry.countries.search_fuzzy(model_place)[0].alpha_3
 
     r = requests.get(f"https://www.geoboundaries.org/gbRequest.html?ISO={country_code}&ADM=ADM0")
     dlPath = r.json()[0]["gjDownloadURL"]
     geoBoundary = requests.get(dlPath).json()
     if not geoBoundary["features"]:
         return MultiPolygon([0])
     geo = geoBoundary["features"][0]["geometry"]
-    return MultiPolygon([Polygon(poly[0]) for poly in geo["coordinates"]])
+
+    return MultiPolygon([Polygon(poly[0]) for poly in geo["coordinates"]]).wkb
 
 
-def add_country_borders_to_model(country_name: str, project, overwrite=False):
+def add_country_borders_to_model(model_place: str, project, overwrite=False, source="gadm"):
+    """
+    Import and add country borders to model.
+    Parameters:
+         *model_place*(:obj:`str`): current model place
+         *project*(:obj:`aequilibrae.project`): currently open project
+         *overwrite*(:obj:`bool`): overwrite existing country borders. Defaults to False.
+         *source*(:obj:`str`): data source to import subdivisions. Takes GADM or GeoBoundaries. Defaults to GADM.
```

Comment:
Gadm is upper case or lower case?  The docstring says one thing, but the default value says another

Reply:
And i see you made it independent below (with the .lower()), but let's make the documentation consistent